src/translation-assistant.js

```diff
@@ -358,38 +358,69 @@ function createTemplate(englishStr, translatedStr, lang) {
     }
 }
 
-// This array is used both in translateMath and normalizeTranslatedMath
-const THOUSAND_SEPARATOR_AS_THIN_SPACE_LOCALES = ['cs', 'fr', 'de',
-      'pt-pt', 'nb', 'bg', 'pl', 'ro', 'nl', 'az', 'sv', 'it', 'hu', 'uk'];
-
 /**
- * Handles any per language special case translations
- * e.g. Portuguese uses `sen` instead of `sin`,
- * many languages use decimal comma etc.
- *
- * The list of all per-locale math notations is in this table:
+ * Locale lists for different math notations, taken from this table:
  * https://docs.google.com/spreadsheets/d/1qgi-KjumcZ6yru19U5weqZK9TosRlTdLZqbXbABBJoQ/edit#gid=0
- *
- * TODO(danielhollas): For now only the obvious cases were implemented.
  * TODO(danielhollas): Need to update this when new langs join translations
+ */
+const MATH_RULES_LOCALES = {
+    'THOUSAND_SEP_AS_THIN_SPACE': ['cs', 'fr', 'de',
+         'pt-pt', 'nb', 'bg', 'pl', 'ro', 'nl', 'az', 'sv', 'it', 'hu', 'uk'],
+
+    'THOUSAND_SEP_AS_DOT': ['pt', 'tr', 'da', 'sr', 'el', 'gr'],
+
+    'NO_THOUSAND_SEP': ['ko', 'ps'],
+
+    'DECIMAL_COMMA': ['cs', 'fr', 'de', 'pl', 'bg', 'nb', 'tr', 'da', 'sr',
+            'ro', 'nl', 'hu', 'az', 'it', 'pt', 'pt-pt', 'sv'],
+
+    'TIMES_AS_CDOT': ['cs', 'pl', 'de', 'nb', 'sr', 'ro', 'hu', 'sv', 'da'],
+
+    'DIV_AS_COLON': ['cs', 'de', 'bg', 'hu', 'uk', 'da'],
+
+    'SIN_AS_SEN': ['it', 'pt', 'pt-pt'],
+};
+
+/**
+ * Handles any per language special case translations
+ * e.g. thousand separators, decimal commas etc.
  *
  * @param {string} math A math expression to translate for locale.
  * @param {string} lang The locale of the translation language.
  * @returns {string} The translated math expression.
  */
 function translateMath(math, lang) {
 
+    const PLACEHOLDER = 'THSEP';
```

Comment:
Could you please add a comment for these function constants, mentioning that these are local to the function, regexs that are used to manipulate expressions?

Let's also lowecase `PLACEHOLDER` please, as it's not a file const (it looks like it should be a file level const, which confuses me every time I see it :)